files/cluster_provision.yml

```diff
@@ -0,0 +1,89 @@
+ - job:
+         name: cluster_provision
+         description: "This is the job to provision a cluster using MrP"
+         project-type: freestyle
+         block-downstream: false
+         concurrent: true
+         node: "master"
+         properties:
+                 - authorization:
+                         hpc-sig-admin:
+                                 - credentials-create
+                                 - credentials-delete
+                                 - credentials-manage-domains
+                                 - credentials-update
+                                 - credentials-view
+                                 - job-build
+                                 - job-cancel
+                                 - job-configure
+                                 - job-delete
+                                 - job-discover
+                                 - job-move
+                                 - job-read
+                                 - job-status
+                                 - job-workspace
+                                 - ownership-jobs
+                                 - run-delete
+                                 - run-update
+                                 - scm-tag
+                         hpc-sig-devel:
+                                 - job-build
+                                 - job-read
+         parameters:
+                 - choice:
+                         name: machine_type
+                         choices:
+                                 - d05
```

Comment:
also, let's use the names we have in the file server: "huawei" and "qualcomm" for these.